.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -124,5 +124,5 @@ body:
           required: true
         - label: I have verified that there isn't [already an issue](https://github.com/mdn/yari/issues) that reports the same bug to avoid creating a duplicate.
           required: true
-        - label: I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mdn/yari/discussions).
+        - label: I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/orgs/mdn/discussions).
```

Comment:
@schalkneethling Should we link here specifically to the [platform section](https://github.com/orgs/mdn/discussions/categories/platform)?

Reply:
> @schalkneethling Should we link here specifically to the [platform section](https://github.com/orgs/mdn/discussions/categories/platform)?

Yeah, that sounds like a good idea. Thank you, @rakleed and @caugner 🎉

Reply:
```suggestion
        - label: I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/orgs/mdn/discussions/categories/platform).
```